realm/src/main/java/io/realm/Realm.java

```diff
@@ -1276,6 +1276,7 @@ boolean hasChanged() {
     public void refresh() {
         checkIfValid();
         transaction.advanceRead();
+        sendNotifications();
```

Comment:
Right now `advanceRead()` returns `void`, but that would be a nice addition once available!